src/main/resources/languages/en/messages.yml

```diff
@@ -145,12 +145,16 @@ guide:
       resourcepack: '&cResourcepack Artist'
       translator: '&9Translator'
 
+  unlock-mode:
+    experience: "Level(s)"
+    currency: "money"
+
 actionbar:
   radiation: '&6Radiation Exposure Level: &c%level%&7/&e100'
 
 messages:
   not-researched: '&4You do not have enough knowledge to understand this. &cYou will need to unlock &f"%item%&f"'
-  not-enough-xp: '&4You do not have enough XP to unlock this'
+  not-meet-requirement: '&4You do not meet the requirement to unlock this'
```

Comment:
should probably call it something like does-not-meet-requirement or something like that

Reply:
maybe `requirement-unfulfilled`? or `unfulfilled-requirement`